src/Model/User/User.php

```diff
@@ -110,7 +110,7 @@ class User extends BaseModel
     private $notes;
 
     /**
-     * @var int|null
+     * @var bool|null
      * @SerializedName("organisation_access_level")
```

Comment:
While current values are `0` or `1` I'm not sure that this should be boolean. @jshah4517 ?

Reply:
This makes more sense as just an int, it's 0` - Manager, `1` - User.